File: app/src/main/java/za/co/woolworths/financial/services/android/models/dto/PromotionImages.java
package za.co.woolworths.financial.services.android.models.dto;

/**
 * Created by dimitrij on 2017/01/12.
 */
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PromotionImages {

    @SerializedName("newImage")
    @Expose
    public String newImage;
    @SerializedName("save")
    @Expose
    public String save;
    @SerializedName("vitality")
    @Expose
    public String vitality;
    @SerializedName("wRewards")
    @Expose
    public String wRewards;
    @SerializedName("freeGift")
    @Expose
    public String freeGift;
    @SerializedName("reduced")
    @Expose
    public String reduced;
}

File: app/src/main/java/za/co/woolworths/financial/services/android/models/dto/PromotionImages.kt
package za.co.woolworths.financial.services.android.models.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by dimitrij on 2017/01/12.
 */
@Parcelize
data class PromotionImages(
    @SerializedName("newImage")
    var newImage: String? = null,
    @SerializedName("save")
    var save: String? = null,
    @SerializedName("vitality")
    var vitality: String? = null,
    @SerializedName("wRewards")
    var wRewards: String? = null,
    @SerializedName("freeGift")
    var freeGift: String? = null,
    @SerializedName("reduced")
    var reduced: String? = null,
    @SerializedName("wList")
    var wList: String? = null,

    ) : Parcelable